omod/src/test/java/org/openmrs/module/idgen/web/search/LogEntrySearchHandlerTest.java

```diff
@@ -82,7 +82,46 @@ public void getSearchConfig_shouldReturnLogEntryByDateGenerated() throws Excepti
 	public void getSearchConfig_shouldReturnLogEntryByGeneratedBy() throws Exception {
 		MockHttpServletRequest req = request(RequestMethod.GET, getURI());
 		req.addParameter("generatedBy", USER_UUID);
+		req.addParameter("comment", "New");
+		SimpleObject result = deserialize(handle(req));
+		Assert.assertEquals(2, Util.getResultsSize(result));
+	}
+
+	@Test
+	public void getSearchConfig_shouldReturnLogEntryByGeneratedByandComment() throws Exception {
```

Comment:
Did you intentionally have some of your "And" s start with capital A while others start with small a?

Reply:
Changed them all to start with "A". Also pushed the failing tests